src/design/Table/index.tsx

```diff
@@ -108,3 +109,19 @@ export const TableRow = React.forwardRef<
   </tr>
 ));
 TableRow.displayName = "TableRow";
+
+export const NoResult = ({
+  message,
+  icon: Icon,
+}: {
+  message: string;
+  icon?: LucideIcon;
+}) => (
+  <div
+    className="flex grow flex-col items-center justify-center gap-1 p-10"
+    data-testid="no-result"
+  >
+    {Icon && <Icon />}
+    <span className="text-center text-sm">{message}</span>
```

Comment:
Could the `message` prop just be `children` here? I'm a big fan of using children whenever it makes sense` <Component>text</Component>` looks a bit more react-ish in comparison to `<Component text="text" />`

Reply:
Good point! I applied this in the last 2 commits.